src/components/ADempiere/Form/VPOS/ProductInfo/productList.vue

```diff
@@ -63,7 +63,11 @@
         align="right"
       >
         <template slot-scope="scope">
-          {{ formatPrice(scope.row.priceStandard) }}
+          {{
```

Comment:
Just can see change for source code with the same functionality, please let it without changes